pkg/kube/controllers/queue.go

```diff
@@ -0,0 +1,168 @@
+// Copyright Istio Authors
```

Comment:
how about putting it under ` pkg/kube/queue`

Reply:
A bit worried about confusion with `pkg/queue`... maybe we can move it after we move everything over?

Reply:
sgtm